sycl/plugins/unified_runtime/ur/adapters/level_zero/memory.cpp

```diff
@@ -1621,7 +1621,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urMemImageCreate(
              (Context->ZeContext, Device->ZeDevice, &ZeImageDesc, &ZeImage));
 
   try {
-    auto UrImage = new _ur_image(Context, ZeImage);
+    auto UrImage = new _ur_image(Context, ZeImage, true /*OwnZeMemHandle*/);
```

Comment:
Perhaps better not to provide a default value of "false"

Reply:
possibly, but the way we have it is OwnNativeHandle is defined in _ur_object here https://github.com/intel/llvm/blob/b9c8516ddd1195501a5914f09c84d7d3f83318a1/sycl/plugins/unified_runtime/ur/adapters/level_zero/common.hpp#L388, so changing that default would affect more than just images. Changing that default could be more risky, so prefer to have it as is now to have a more contained change.

Reply:
That is OK. If we ever refactor this then perhaps the better default would "true".